editor/editor_properties_array_dict.cpp

```diff
@@ -590,6 +590,9 @@ bool EditorPropertyArray::_is_drop_valid(const Dictionary &p_drag_data) const {
 
 			for (String at : allowed_type.split(",")) {
```

Comment:
You could do this instead:
```suggestion
			for (String at : allowed_type.split(",", false)) {
```